config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # root "users#index"
+  get 'users' => 'users#index'
+  get 'users/:id' => 'users#show'
+  get 'users/:user_id/posts/' => 'posts#index'
+  get 'users/:user_id/posts/:id' => 'posts#show'
```

Comment:
- [ ] [OPTIONAL] You did great adding routes but I think using resources would make them cleaner and more readable 

```ruby

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```